app/models/ebean/Report.java

```diff
@@ -92,8 +92,11 @@
     @JoinColumn(name = "report_schedule_id")
     private ReportSchedule schedule;
 
-    @Column(name = "timeout_nanos")
-    private long timeoutNanos;
+    @Column(name = "render_timeout_nanos")
```

Comment:
Did you create a previous migration for this? Wouldn't this create duplicate columns?